gui/resultstree.cpp

```diff
@@ -289,32 +289,9 @@ QStandardItem *ResultsTree::addBacktraceFiles(QStandardItem *parent,
 
 QString ResultsTree::severityToTranslatedString(Severity::SeverityType severity)
 {
-    switch (severity) {
-    case Severity::style:
-        return tr("style");
-
-    case Severity::error:
-        return tr("error");
-
-    case Severity::warning:
-        return tr("warning");
-
-    case Severity::performance:
-        return tr("performance");
-
-    case Severity::portability:
-        return tr("portability");
-
-    case Severity::information:
-        return tr("information");
-
-    case Severity::debug:
-        return tr("debug");
-
-    case Severity::none:
-    default:
+    if ((severity <= Severity::none) || severity > Severity::debug)
         return QString();
-    }
+    return tr(Severity::toString(severity).c_str());
```

Comment:
I doubt that this works.

Reply:
With "it works" it means that Qt Linguist will be able to extract what possible return values Severity::toString has (by looking at this code).. and then show the variations in the Qt Linguist interface and those will be used properly.

Reply:
From Qt, signature of [tr() or translate()](https://doc.qt.io/qt-5/qcoreapplication.html#translate) is char* returning QString. Severity::toString provides std::string which provides c_str(). I don't think this should cause a problem.